### DISCUSSION TITLE
i dropped something . . . it's my expectations for public school education

### DISCUSSION THREAD

**Discussion by vcta99** [albertkun/21S-ASIAAM-191A#83] [Past Thinking Caps]
While I think that education and increased accessibility will further promote participatory mapping and critical cartography, I'm not sure if I believe it is a realistic goal that can be implemented. I think that I, like many others in this class, had never heard the term "critical cartography" until they entered UCLA. The K-12 public school system is so backwards that much of my college experience was spent unlearning what was previously taught to me for twelve years straight (hello, internalized racism). 

If I were to take several steps back and zoom the lens back, I would more broadly reform the way American history is taught in general. I say that like it would be easy, but it would not be. I do think, however, that it is the most effective way to start. Once a framework of decolonization is developed, it can be applied to other subjects that already exist within the system (English, Environmental Science, etc.), and hopefully to the system itself. 

This response may not have been super focused on participatory mapping specifically, but I think that, at it's core, critical cartography serves to decenter white supremacy. And I do believe that tools to challenge white supremacy and empower communities should be made more accessible, much earlier in life. All this is just to say that I think the public school system are still stuck on the question "does racism exist? explain why or why not in 500 words", and I think it needs to radically move past that before topics like critical cartography are introduced. I do believe that we can get there one day, but at the current rate of where things are going, that day may be a ways away. 

Here's a screenshot of a twitter post I thought was relevant. 

![image](https://user-images.githubusercontent.com/52229582/114350701-69b92e80-9b1e-11eb-8b9a-4263d15c0f70.png)

**Comment by jason-muljadi** [comment]
Very true! A lot of K-12 curriculum centers a lot through eurocentric/Western lenses and it's so easy to get into that mentality. Not to mention that American nationalism becomes a core part of the curriculum so, oftentimes, America is often portrayed as the "good guy" in a lot of the geopolitical issues that we face today. 

I very much agree with having an interdisciplinary lens when it comes to core classes in early education. Kids are smart; it would be asinine to treat them as if they cannot grasp at least the preliminary concepts of "difficult topics." 

Another issue is also the communities that surround K-12 education. It is one step in order to dismantle the curriculum and replace it with one that does value equity and less of the harmful, negotiable rhetoric that dehumanizes people of color. Ensuring that communities are also educated about these issues is as equally as important, as a community can have an affect on propagating viewpoints, especially if an individual is community-centric.

**Comment by alizezila** [comment]
Thanks for sharing!! I absolutely agree that critical cartography can serve to de-center white supremacy and core nations' legacy of colonialism/imperialism. A simple change can go a long way. I wonder how different my understanding of the world and processes of development would be if my elementary school teachers had a map that showed the appropriate sizes of Africa and Greenland, unlike the common mercator map.

**Comment by okcirE** [comment]
Through reading your post, I got reminded of the critical race theory which states that that race is not a biologically grounded feature of physically distinct subgroups of human beings but a socially constructed (culturally invented) category that is used to oppress and exploit people of colour. And how the education in CA is implementing ethnic studies course as a main elective in order to graduate high school. This class is aimed to dissect American history and society through a CRT lens. However, they have yet to start creating the curriculum and it is worth noting to be careful when structuring the learning objectives because of bias. I believe in the correct and positive manner our education system can educate student that dissecting racial bias is complex.

**Comment by brillianz** [comment]
While many in the thinking caps have (rightfully) noted the unlearning that they did at UCLA following their public education, I would like to caution that the university is by no means the only place where learning and education can be found. I have found several new approaches and subjects through informal learning spaces and "knowledge shares," both before, and increasingly after the pandemic through the accessibility of remote learning. The advocacy initiatives that our professor has mentioned he has participated in are just some examples of how this can be manifested. Beyond the accessibility that this provides to a broader audience, it also enables everyone involved in the community initiative to work horizontally towards a particular goal that is at hand. 

During my time as an organizer for Save CA Ethnic Studies in 2019, several campaigns were made to assure that a _critical_ ethnic studies curriculum would be in place for interested California high schools. Beyond CRT as a theoretical approach, the main concern of the curriculum would fall under the interest of including the four main racialized groups in the country: Black people, Chicanos/Latinos, Native Americans, and Asian Americans. The relationships that critical educators had throughout the state ensured that we would have the leverage necessary to campaign towards this goal with sufficient numbers whenever the occasion arose. We "mapped" these connections through tables that listed out people and their affiliations by their geographic locations in the state. This worked to our favor whenever there was a state committee meeting in place and we wanted to show up to support.
